main.go

```diff
@@ -91,6 +161,8 @@ func (k *kubeRulesLoader) getTenantRuleGroups(prometheusRules []*monitoringv1.Pr
 type tenantRulesSyncer interface {
 	initOrReloadObsctlConfig() error
 	setCurrentTenant(tenant string) error
+	obsctlLogsAlertingSet(rules lokiv1beta1.AlertingRuleSpec) error
```

Comment:
Seems like some of the methods are a bit repetitive due to split in `lokiv1beta1.AlertingRuleSpec` and `lokiv1beta1.RecordingRuleSpec`. I wonder if we can convert these to `monitoringv1.PrometheusRuleSpec` somehow and unify the methods? 
I think Loki rules are compatible with prometheus afaik. Wdyt? 🙂